Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -616,7 +618,9 @@ WarpXOpenPMDPlot::WriteOpenPMDParticles (const amrex::Vector<ParticleDiag>& part
       GeometryFilter const geometry_filter(particle_diags[i].m_do_geom_filter,
                                            particle_diags[i].m_diag_domain);
 
-      if (! isBTD) {
+      if (isBTD || use_pinned_pc) {
```

Comment:
Side-note for a potential follow-up PR:
There is some similar code here:
https://github.com/ECP-WarpX/WarpX/blob/development/Source/Diagnostics/FlushFormats/FlushFormatPlotfile.cpp
Should we try to unify these two functions before they diverge too much?

Reply:
That was something that I was considering. I'll test this and see if the writing out of the particles will work with plotfiles (instead of only openpmd).